src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/child-place-of-birth.js

```diff
@@ -0,0 +1,116 @@
+import { genericSchemas } from '../../../generic-schema';
+import { childInfo } from '../child-information/helpers';
+import { childStatusDescription } from './childStatusDescription';
+import { isChapterFieldRequired } from '../../../helpers';
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+import _ from 'lodash/fp';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    childrenToAdd: {
+      type: 'array',
+      minItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          state: genericSchemas.genericTextInput,
+          city: genericSchemas.genericTextInput,
+          childStatus: {
+            type: 'object',
+            properties: {
+              biological: genericSchemas.genericTrueFalse,
+              adopted: genericSchemas.genericTrueFalse,
+              notCapable: genericSchemas.genericTrueFalse,
+              stepchild: genericSchemas.genericTrueFalse,
+              dateBecameDependent: genericSchemas.date,
+            },
+          },
+          'view:marriageTypeInformation': {
+            type: 'object',
+            properties: {},
+          },
+          childPreviouslyMarried: {
+            type: 'string',
+            enum: ['Yes', 'No'],
+            default: 'No',
+          },
+          childPreviousMarriageDetails: {
+            type: 'object',
+            properties: {
+              dateMarriageEnded: genericSchemas.date,
+              reasonMarriageEnded: {
+                type: 'string',
+                enum: ['Divorce', 'Death', 'Annulment', 'Other'],
+                default: 'Divorce',
+              },
```

Comment:
It's a little ambiguous in the mockups, but if a user selects 'Other', then there needs to be a text field that expands below that asks for a brief explanation. [Similar to the type of marriage workflow](https://xd.adobe.com/view/e185627d-0f83-4762-747d-d646b4557713-97b4/screen/afeede90-907d-401d-88c3-4160830fb253/Apply-Dependents-Marriage-Info-1). I confirmed this very thing with Jim when I was putting together the `add spouse` workflow. 

You could probably just re-use most of [this](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/686c-674/config/chapters/report-add-a-spouse/current-marriage-information/currentMarriageInformation.js#L50)